### ISSUE TITLE
TFLite conversion change model weights

### ISSUE THREAD

**Issue by wshilin** [tensorflow/tensorflow#32033]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution : Windows Server 2016
- TensorFlow installed from source
- TensorFlow version: 1.14
- Python version:3.5

**Describe the current behavior**
I convert .pb file to .tflite file .Then I found the model weights changed in .tflite file.
My CNN model just some separable convolution layers.In fact, I just replaced the ordinary convolution layer of VGG16 with the separable convolution layer. 
All the pointwise convolution  weights change in .tflite file ,while all the depthwise convolution weights remain the same in two files.
As you can see below as an example.

### **pointwise convolution weights in .pb:**
![1566965436(1)](https://user-images.githubusercontent.com/4417111/63825376-2cc21780-c98d-11e9-8802-27b67e615c19.png)

### **pointwise convolution weights in .tflite**
![1566965467(1)](https://user-images.githubusercontent.com/4417111/63825409-46fbf580-c98d-11e9-9efa-993f1833237c.png)

**Code to reproduce the issue**
### **I used this code below to freeze model weights and saved as .pb file.**
![1566965770(1)](https://user-images.githubusercontent.com/4417111/63825655-2ed8a600-c98e-11e9-8e01-62420e930fac.png)

### **I used this code below to convert .pb file to .tflite file.**
![1566965842(1)](https://user-images.githubusercontent.com/4417111/63825687-49128400-c98e-11e9-8853-c7b376b11c92.png)

### **I used this code below to print weights in .pb file and .tflite file.**
![1566965966(1)](https://user-images.githubusercontent.com/4417111/63825722-69dad980-c98e-11e9-8847-fae9f5e50884.png)

I don't konw why this problem occur.

**Comment by jiarenyf** [comment]
https://github.com/tensorflow/tensorflow/issues/31359

**Comment by liyunlu0618** [comment]
We do optimizations like folding batchnorm constants into the conv filter when converting tf models to tflite. It's expected to change the values of the tensor. Please compare whether the two models generate different output. If not it's a real bug. Feel free to reopen if you find that's the case. Thanks!
